aptos-move/move-examples/defi/aptoswap/core/sources/liquidity_pool.move

```diff
@@ -0,0 +1,417 @@
+module pool_manager::liquidity_pool {
+    use aptos_framework::fungible_asset::{Metadata, FungibleStore, FungibleAsset, deposit};
+    use aptos_framework::object::{Object, ExtendRef, generate_signer_for_extending, generate_extend_ref};
+    use aptos_framework::fungible_asset;
+    use aptos_framework::object;
+    use aptos_framework::primary_fungible_store;
+    use aptos_std::comparator;
+    use aptos_std::math128;
+    use aptos_std::math64;
+    use aptos_std::smart_table::SmartTable;
+    use aptos_std::smart_table;
```

Comment:
nit: Merge with line 10. We should probably file a request in the intellij Move plugin repo to request automatic import merging

Reply:
yeah, I never noticed this cuz it's folded.